melodic/distribution.yaml

```diff
@@ -4551,6 +4551,16 @@ repositories:
       url: https://github.com/ros/kdl_parser.git
       version: melodic-devel
     status: maintained
+  keypress_monitor:
+    doc:
+      type: git
+      url: https://repo.ijs.si/msimonic/keypress_monitor.git
```

Comment:
It looks like the license listed in the package.xml is GPLv2.  Can you please also add a COPYING file to the top-level as specified in the GPL license?

Reply:
@clalancette Thanks for pointing that out. I added the copying file in https://repo.ijs.si/msimonic/keypress_monitor/-/commit/4b1616e147cfc8573ac3c8f8bfd0731d64e1cf98